### DISCUSSION TITLE
VSCode launches in very low-res

### DISCUSSION THREAD

**Discussion by andrerossellm** [prasanthrangan/hyprdots#66] [Q&A]
Hi! First of all, thank you for the amazing work. 
I've just installed the rice using the script on a fresh base arch install and I'm loving it! I have a weird issue which I've not been able to solve. My VSCode windows are all blurry/low-res no matter how I launch the application. 
![image](https://github.com/prasanthrangan/hyprdots/assets/92490116/85cd8e8d-a5aa-479d-8aec-7906cc5740a3)
I can avoid it by launching it forcing Wayland with two flags as mentioned over [here](https://www.reddit.com/r/swaywm/comments/n8dymo/vs_code_finally_works_natively_in_wayland/), but this also disables the blur effect from the dotfiles. 
![image](https://github.com/prasanthrangan/hyprdots/assets/92490116/e70db0dc-6dc5-4d43-8254-3243bf272424)

Have any of you encountered this? Does anybody know how the author forces blur on VSCode so I can try to replicate it while using the flags? 
Thanks!

**Comment by Gedankenn** [comment]
I had the same problem,  my monitor is 4k and the vscode didn't liked any scaling for it.
My solution was to set the resolution and the scale for the monitor in the monitor config

**Reply by andrerossellm** [reply]
I'll try it out later, thanks for the tip!
By the way, would you happen to know how he manages  to get blur on the firefox window? I'm trying to move to other browser and want to apply the same effect

**Reply by T-Crypt** [reply]
Check in 

Hyprdots/Configs/.config/hypr/windowrules.conf

Line 13 is what you are looking for:

- windowrulev2 = opacity 0.90 0.90,class:^(firefox)$

**Reply by sudobrat** [reply]
how do i set monitor config?

**Reply by T-Crypt** [reply]
@GeekWolf007 

Settings for your monitor config are in:

/home/user/.config/hypr/monitors.conf 

This will set the user preferences for your monitor over the main Hyprland Config.

**Reply by sudobrat** [reply]
but the monitor .config file is empty

**Reply by T-Crypt** [reply]
@GeekWolf007 As it should be, the monitors.conf is set to override the hyprland conf.

Try taking a look at the hyprland monitor wiki, fairly simple to add monitors. 

rule for plugging in random monitors >>> monitor=,preferred,auto,1

-- https://wiki.hyprland.org/Configuring/Monitors/
